index.js

```diff
@@ -488,7 +488,7 @@ function hasEncryptionScheme(mediaKeySystemAccess) {
   // If supported by the browser, the encryptionScheme field must appear in
   // the returned configuration, regardless of whether or not it was
   // specified in the supportedConfigurations given by the application.
-  if (firstCapability['encryptionScheme'] !== undefined) {
+  if (firstCapability && firstCapability['encryptionScheme'] !== undefined) {
```

Comment:
This is a simple enough fix, and I see no reason not to accept it.  But how can Edge resolve a mediaKeySystemAccess if PlayReady isn't supported?  How can it return a configuration if there are no capabilities in it?

Reply:
Or perhaps I misread the issue.  Is it that PlayReady *is* supported by the browser, but this polyfill makes it appear unsupported by throwing an exception?

Reply:
yes, exactly
PlayReady DRM is supported by the browser and 
```
navigator.requestMediaKeySystemAccess('com.microsoft.playready', config)
```
will return MediaKeySystemAccess instance but with an empty array of videoCapabilities

So, actually it is a strange behaviour of MS Edge which is reproduced only on specific versions of a browser